Ascend.java

```diff
@@ -3,5 +3,14 @@
 public class Ascend {
 	public static void main(String[] args) {
 		// Replace this comment with your code
+		int lim = Integer.parseInt(args[0]);
+		int a= (int) (Math.random() * lim);

```

Comment:
Before and after "=" sign there should be a space...
It seems like this is the only place it happened so I guess it was a mistake, but make sure to go through your code and check for Java styling guidlines!